docs/cross-chain/REBALANCING_FLOW.md

```diff
@@ -98,25 +98,27 @@ sequenceDiagram
 
 #### MEV Protection in Cross-Chain Operations
 
-The FleetCommander implements cooldown-based MEV protection:
+The FleetCommander implements withdrawal fee-based MEV protection:
 
 **Deposit Phase:**
-- User deposits trigger timestamp recording via `_updateLastDepositTimestamp()`
-- Cooldown period starts immediately after deposit confirmation
+- User deposits are processed immediately without restrictions
+- No cooldown periods or timestamp tracking required
```

Comment:
I keep trying to understand why we are referring to things that are not going to make it to the final version. I take that the cooldown is a discarded idea, so I wouldn't mention it here (it can be mentioned in the MEV protection description section to inform readers that we considered the idea). Just because anybody reading this would wonder what cooldown periods or timestamp tracking are and why they are not needed in the context of deposits